test/shared/visu/bpmn-page-utils.ts

```diff
@@ -224,7 +223,15 @@ export class PageTester {
 
       if ('fill' in styleUpdate) {
         const fill = (<ShapeStyleUpdate>styleUpdate).fill;
-        fill.color && (url += `&style.api.fill.color=${fill.color}`);
+
+        if (typeof fill.color !== 'string') {
+          fill.color.startColor && (url += `&style.api.fill.color.startColor=${fill.color.startColor}`);
```

Comment:
**suggestion:** introduce a fillColor constant as fill.color is called a lot of times